crates/sui-cost/src/cost.rs

```diff
@@ -27,17 +27,18 @@ pub const MAX_TX_GAS: u64 = 1_000_000_000;
 // Fixed costs: these are charged regardless of execution
 //
 // This is a flat fee
-pub const BASE_TX_COST_FIXED: u64 = 1_000;
+pub const BASE_TX_COST_FIXED: u64 = 10_000;
```

Comment:
How are numbers chosen for each of the constant change?

Reply:
These are actually the same numbers that was used in gas.rs
I just put them in here since we're now exporting the constants. This way nothing breaks over the weekend.
The true numbers are being discussed with Econ and will be phased in during the week.